setup/ci/gcb.md

```diff
@@ -20,35 +22,37 @@ You can enable the API with the following `gcloud` command:
 gcloud services enable cloudbuild.googleapis.com
 ```
 
-## Configure Spinnaker to Listen for Google Cloud Build Pub/Sub Notifications
-
 Google Cloud Build sends [Build Notifications](https://cloud.google.com/cloud-build/docs/send-build-notifications)
-when the state of your build changes. You can create a pipeline trigger to invoke a pipeline based
-on the status of your build.
+when the state of your build changes.  Spinnaker subscribes to these pub/sub messages so that in can track the status
+of builds it has initiated in a GCB stage and to trigger pipelines based on build status changes.
 
-1. Create a Subscription object for the `cloud-builds` topic in your project:
+Create a Subscription object for the `cloud-builds` topic in your project:
```

Comment:
Actually one random nit: should we either append a colon to all enumerated instructions that are followed by commands, or none of them?

Reply:
Good point.  @dorbin , any preference or convention with regards to this?

Reply:
I say include it, with all.

Reply:
Updated all instructions to be preceded by colons (as none seemed to fit the case for needing an elipsis)